test/regression/expected/type_support.out

```diff
@@ -308,6 +308,27 @@ SELECT * FROM json_tbl;
  {}
 (4 rows)
 
+-- BLOB
+CREATE TABLE blob_tbl(a bytea);
+INSERT INTO blob_tbl SELECT CAST(a as bytea) FROM (VALUES
+    ('\x'),
+    ('\x110102030405060708090a0b0c0d0e0f'),
+    (''),
+    ('\x00'),
+    ('\x07'),
+    (NULL)
+) t(a);
+SELECT * from blob_tbl;
+                                                                 a                                                                  
+------------------------------------------------------------------------------------------------------------------------------------
+ \x
+ \x5c7831315c7830315c7830325c7830335c7830345c7830355c7830365c7830375c7830385c7830395c7830415c7830425c7830435c7830445c7830455c783046
+ \x
+ \x5c783030
```

Comment:
> currently the output from select * from blob_tbl is an escaped version of inserted rows, is this fine or do we need to change that?

Okay, this needs to change. All the non empty bytea results are wrong. When using postgres execution the it instead gives the following rows:
```
 \x
 \x110102030405060708090a0b0c0d0e0f
 \x
 \x00
 \x07

 ```

It seems like you're encoding the string as hex somewhere as an additional time, because `5c783030` is `\x00` in ASCII, i.e. including the `\` and `x` characters. So I think when converting between from DuckDB to PG type you're encoding the string representation of the type instead of the raw bytes.

Reply:
Found it, converting it to string was trying to convert those bytes to string

Reply:
Changed GetValue to GetValueUnsafe for getting raw bytes in string_t